master/buildbot/test/unit/test_plugins.py

```diff
@@ -71,7 +71,7 @@ class ITestInterface(IPlugin):
     """
     test interface
     """
-    def hello(name):
```

Comment:
Generally interfaces don't list `self`, but that confuses pylint.  It doesn't hurt to have `self`, so this is fine.